data/mods/Magiclysm/recipes/magic_tools.json

```diff
@@ -44,14 +44,27 @@
     "result": "silver_infuser",
     "category": "CC_ENCHANTED",
     "subcategory": "CSC_ENCHANTED_TOOLS",
-    "skill_used": "fabrication",
-    "skills_required": [ "spellcraft", 3 ],
+    "skill_used": "spellcraft",
```

Comment:
no. put this back. making infusers should not exercise your spellcraft. same for the gold one.